boggle_solver.js

```diff
@@ -34,13 +34,15 @@ class TrieTree {
 		let node = this.root;
 		for (let i = 0; i < word.length; i++) {
 			// check to see if character node exists in children.
+			console.log(word[i]);
 			if (!node.children[word[i]]) {
```

Comment:
!node.children[word[i]] could be a variable in the previous line ( like let ifCharNodeExist = !node.children[word[i]] ) for readability